source/_integrations/caldav.markdown

```diff
@@ -14,7 +14,7 @@ The `caldav` platform allows you to connect to your WebDAV calendar and generate
 
 ## Prerequisites
 
-You need to have a CalDAV server and credentials for it. This integration was tested against [Baikal](http://sabre.io/baikal/) but any integration complying with the RFC4791 should work. [Nextcloud](https://nextcloud.com/) and [Owncloud](https://owncloud.org/) work fine.
+You need to have a CalDAV server and credentials for it. This integration was tested against [Baikal](http://sabre.io/baikal/) but any integration complying with the RFC4791 should work. [Nextcloud](https://nextcloud.com/) and [Owncloud](https://owncloud.org/) work partially, full day event or mulitple days events ane not synchronised and don't show up in Home Assistant.
```

Comment:
This extension is confusion. It is an issue with Nextcloud and Owncloud specifically? Or a caldav limitation? Or is this a bug in the Caldav integration?

Reply:
it's a bug in Caldav integration I think ! I use Caldav sync with my NC without any problems like in NC Caldav integration !

Reply:
If it's a bug with the integration, it should be reported and fixed; not documented.

We don't document bugs.

Reply:
I don't want to be rude with anyone but that bug exists since ages and has never been fixed so that's why I thought it was better to document it (regular questions in community about it and explain each time it's buggy whatever description of integration states....

Reply:
The review comment stays the same. If you experience a bug, open up an issue.